portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Leonard Zhang's Photo Gallery</title>
+    <link rel="stylesheet" href="gallery_style.css">
+    <script src="change_picture_script.js"></script>
+  </head>
+  <body>
+    <p id="return-link"><a href="index.html"><< Go back to the main portfolio page</a></p>
```

Comment:
I'd suggest using HTML entity tags here and below instead of literal `<` and `>` characters, which might get confused with HTML tag syntax. The entity tags for those are `&lt;` and `&gt;`, respectively. They make it clear that yes, you want to show a literal less-than or greater-than sign in the document, and the browser will render them as such.

Reply:
Oh I forgot the existence of &lt and &gt. I will fix this.